src/connectedmachine/azext_connectedmachine/generated/_params.py

```diff
@@ -0,0 +1,105 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_three_state_flag,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import (
+    get_default_location_from_resource_group,
+    validate_file_or_dict
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('connectedmachine machine list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('connectedmachine machine show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('machine_name', options_list=['--name', '-n', '--machine-name'], type=str, help='The name of the '
+                   'hybrid machine.', id_part='name')
+
+    with self.argument_context('connectedmachine machine delete') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('machine_name', options_list=['--name', '-n', '--machine-name'], type=str, help='The name of the '
+                   'hybrid machine.', id_part='name')
+
+    with self.argument_context('connectedmachine machine-extension list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('machine_name', type=str, help='The name of the machine containing the extension.')
+        c.argument('expand', type=str, help='The expand expression to apply on the operation.')
+
+    with self.argument_context('connectedmachine machine-extension show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('machine_name', type=str, help='The name of the machine containing the extension.', id_part='name')
+        c.argument('name', type=str, help='The name of the machine extension.', id_part='child_name_1')
+
+    with self.argument_context('connectedmachine machine-extension create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('machine_name', type=str, help='The name of the machine where the extension should be created or '
+                   'updated.')
+        c.argument('name', type=str, help='The name of the machine extension.')
```

Comment:
Add `options_list=['--name', '-n']` for `name` arguments in all below commands as well.

Reply:
The code is generated. How would I do this in a generated way?

Reply:
@qiaozha can we configure the options, probably using [alias](https://github.com/Azure/autorest.az/blob/master/doc/02-customizations.md#setting-parameter-alias)?

Reply:
Ive added the change manually in this PR and in https://github.com/Azure/azure-rest-api-specs/pull/10795